_sass/base/_layout.scss

```diff
@@ -20,16 +20,17 @@ a {
 
 header {
 	font-family: $heading-font-family;
+	font-size: 1.2rem;
+	font-weight: 400;
 	padding-top: 0.25em;
 	margin-top: 10px;
-  padding: 0px 10px;
+ 	padding: 0px 10px;
```

Comment:
Hey Steffan, thanks for the comment. Yes, I agree. The unity then doesn't make any difference. We then stick to just '0' from now on!